### PR TITLE
Added action tests

### PR DESCRIPTION
**Pull request by JR-1991**
* Added tests for the dataverse action
    * `GET` - `api/info/version`
    * `GET` - `api/metadatablocks`
    * `POST` - `api/dataverses/root`
* Removed redundant files and steps
    * unused scripts
    * action steps that call scripts 
* Updated `README.md` and added tests badge

✨ [Successful action run](https://github.com/gdcc/dataverse-action/actions/runs/7066526622/job/19238547751)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+
+This will create a Dataverse instance with the default configuration. The action will output the API token and base URL of the instance, which can be used to interact with the instance. Here is an example on how to re-use the API token and base URL in a subsequent step:
+
+```yaml
+name: How to re-use outputs
+env:
+    API_TOKEN: ${{ steps.dataverse.outputs.api_token }}
+    BASE_URL: ${{ steps.dataverse.outputs.base_url }}
+run: |
+    my-app --api-token $API_TOKEN --base-url $BASE_URL
```
Comment (poikilotherm) [inline README.md:25]:
```suggestion
    my-app --api-token "${{ env.API_TOKEN }}" --base-url "${{ env.BASE_URL }}"
```

**Review by poikilotherm — APPROVED** [review]
Looks good to me. Just one small suggestion, as I think the Syntax fornusing the env variable in the run example is not valid.

**Comment by JR-1991** [comment]
@poikilotherm, thank you for your feedback and for pointing out the example. Commited your proposed changes to the PR. 

I have also extended the outputs of the action to expose the Dataverse version. PyDataverse tests had the version hard-coded, but now with this addition, multiple versions can be tested. 🙌

**Comment by poikilotherm** [comment]
Thank you @JR-1991 for the PR, just merged it!

**Pull request merged**